### DISCUSSION TITLE
Process For Accessing Discord?

### DISCUSSION THREAD

**Discussion by Sortova** [gnuton/asuswrt-merlin.ng#643] [General]
Hello!

On the main page of the project it lists a link for "Chat" which points to a Discord server (https://discord.gg/pm7q6KFSzp). Weeks ago I joined and clicked on the post in the "verify_here" channel but I have yet to be verified. 

I also donated 200€ to the project, or at least I think I did since I never got a confirmation that it was received. I am eager to find out when the major security issue will be addressed so I can turn NAT back on, and I'm trying to be patient but this is becoming a little frustrating.

**Comment by gnuton** [comment]
Hi @Sortova 
DISCORD - 
The verification process is automatized by a bot, so as soon you clicked the "verified" you should get permissions to see other channels.
The process seems to work for me since i do see new people regularly posting in the "welcome" channel.

DONATION -
I have just checked the donation and indeed it was received. Thanks a lot. If you are not happy I would be okay to give it back it in any case. just ping me.

SECURITY 
As for the major security issues, they have been already addressed in the latest 388 prebuilds. Feel free to reach me via chat for any question on discord. In general even after the fix if you care about security I would suggest you to not expose any web service to the web and just keep a VPN gateway open. The VPN is safe and safer. Please evaluate this in any case as an option.

Please ping me via chat per any question.

**Reply by Sortova** [reply]
Hey! A couple of notes.

**Discord**: the bot _does not work_ if you are accessing Discord via the browser. I went to Discord on my mobile, unchecked and then checked the verify box again, and all the channels showed up. Yay!

**Donation**: Please keep it. Even if I went back to stock I have gotten more value out of the project than I paid. I spent 20 years running an open source project called OpenNMS, which did network monitoring. I still use it and so SNMP support is very important to me and is missing from the stock firmware.

**Security**: I have gigabit fiber at my house so I end up serving some features to the world at large (I have a weather station that the neighborhood likes to watch) which is why I use NAT. I also have a somewhat complex network with six nodes and one of them is Ethernet backhaul by a wireless bridge. Five are RT-AX95Q (XT8) version 1 and the newest node is a version 2 (which is not yet supported). I installed the latest alpha on the main router node and ran into an issue that some of my simple IoT devices kept dropping off the network. The screenshot below shows the red lines where my garage camera kept dropping. I downgraded back to .7 and the problem went away (all of the green after that shows uptime). Now part of the issue may be that I need to install the same software on all the other nodes but since it was an alpha I decided to wait for the production release, but if you think it would help I would give it a shot. It's just a lot of work (grin)

<img width="1137" alt="cool-pic 2024-07-15 at 07 54 05" src="https://github.com/user-attachments/assets/3bec58d5-473d-4b2e-ae6d-4a509c3efd65">

When I first started running Merlin in early 2023 is was sometimes a little unstable, but it has been rock solid for me for the last year or so. Thanks for doing this.

**Reply by gnuton** [reply]
@Sortova hey. do you have logs about this issue? alpha should work with previous versions too.

**Reply by Sortova** [reply]
I just reloaded alpha and I keep seeing logs like this:

Jul 28 14:28:46 roamast: eth4: disconnect weak signal strength station [12:7b:82:88:58:83]
Jul 28 14:28:46 roamast: eth4: remove client [12:7b:82:88:58:83] from monitor list
Jul 28 14:28:50 wlceventd: wlceventd_proc_event(645): eth4: Deauth_ind 12:7B:82:88:58:83, status: 0, reason: Previous authentication no longer valid (2), rssi:-80
Jul 28 14:28:50 wlceventd: wlceventd_proc_event(645): eth4: Deauth_ind 12:7B:82:88:58:83, status: 0, reason: Disassociated due to inactivity (4), rssi:-80
Jul 28 14:28:51 wlceventd: wlceventd_proc_event(662): eth4: Disassoc 12:7B:82:88:58:83, status: 0, reason: Disassociated because sending station is leaving (or has left) BSS (8), rssi:0
Jul 28 14:32:32 wlceventd: wlceventd_proc_event(685): eth5: Auth 50:E9:DF:7C:F3:62, status: Successful (0), rssi:0
Jul 28 14:32:33 wlceventd: wlceventd_proc_event(662): eth4: Disassoc 50:E9:DF:7C:F3:62, status: 0, reason: Disassociated because sending station is leaving (or has left) BSS (8), rssi:0
Jul 28 14:32:33 wlceventd: wlceventd_proc_event(695): eth5: ReAssoc 50:E9:DF:7C:F3:62, status: Successful (0), rssi:-58
Jul 28 14:32:33 wlceventd: wlceventd_proc_event(662): eth4: Disassoc 50:E9:DF:7C:F3:62, status: 0, reason: Disassociated because sending station is leaving (or has left) BSS (8), rssi:0
Jul 28 17:03:23 wlceventd: wlceventd_proc_event(645): eth4: Deauth_ind B0:C5:54:0E:3D:36, status: 0, reason: Previous authentication no longer valid (2), rssi:-88
Jul 28 17:03:23 wlceventd: wlceventd_proc_event(645): eth4: Deauth_ind B0:C5:54:0E:3D:36, status: 0, reason: Previous authentication no longer valid (2), rssi:-88
Jul 28 17:42:09 acsd: eth4 received event: tx pkt delay suddently jump
Jul 28 17:42:13 roamast: [EXAP]Deauth old sta in 0 0: 44:61:32:78:3A:41
Jul 28 17:42:13 roamast: eth4: disconnect weak signal strength station [44:61:32:78:3a:41]

If I downgrade the disconnects go away.

**Reply by Sortova** [reply]
<img width="696" alt="cool-pic 2024-07-28 at 18 17 49" src="https://github.com/user-attachments/assets/0176836c-e79e-4a14-9635-9c356ac961bc">

**Reply by gnuton** [reply]
Can you ssh on the router and get an nvram get all  I'm bot cases and run a diff?

**Reply by Sortova** [reply]
I can ssh into the router. (grin)

Can you send me the commands you'd like me to run?

**Reply by gnuton** [reply]
sure you can run the following command in merlin and also in the stock firmware (the one that works)

```
nvram show > nvram_vars.txt
```

then send me both txt files and I will do the diff.. or you can try to do it if you like. :D

**Reply by Sortova** [reply]
Just an update - after this latest upgrade to alpha1 I am seeing a few disconnect/reconnect messages but nothing like before. (shrug)

**Reply by gnuton** [reply]
Hi Sortova,
what do you mean wih"nothing like before"? disconnect/reconnect messages are usually debug messages which you see when ASUS builds their prebuild and leave them enabled for some reasons.
Disconnect and reconnect are usually normal and they may come from devices saving energy.

**Reply by Sortova** [reply]
I really don't know. The first time I installed the alpha1 release I had a number of devices lose and regain connection. I mean, I could almost count on it after the upgrade.

But I've been running it since Monday and it is as stable as can be. No issues.

**Reply by gnuton** [reply]
The diff doesn't have any particular change in the config.
I will most probably release the beta this weekend

**Reply by Sortova** [reply]
I think that's safe.
